package.json

```diff
@@ -118,7 +118,7 @@
     "@types/kerberos": "^1.1.2",
     "@types/minimist": "^1.2.1",
     "@types/mocha": "^9.1.1",
-    "@types/node": "20.x",
+    "@types/node": "^20.14.12",
```

Comment:
Curious why you did change this, might make it a bit harder to keep track of node versions on Electron updates where node.js versions can change.

cc @deepak1556

Reply:
The new typescript version introduces a typings incompatibility with some older typings. Let me see if I can revert the `package.json` change while still picking up the new typings

Reply:
Ah ok, well not sure its really an issue, I only wanted to make sure Deepak gets to see this.

Reply:
Probably I had the opposite thinking, given more frequent minor node.js version updates I wanted to push the types ahead to avoid frequent package.json changes but I can see how that backfires. Happy to pin the version if that is preferred.